components/feature/sitepermissions/src/test/java/mozilla/components/feature/sitepermissions/db/SitePermissionEntityTest.kt

```diff
@@ -25,6 +25,7 @@ class SitePermissionEntityTest {
             bluetooth = ALLOWED,
             autoplayInaudible = BLOCKED,
             autoplayAudible = NO_DECISION,
+            mediaKeySystemAccess = NO_DECISION,
```

Comment:
I think we need to verify this value is correct converted by asserting similar to what we are doing with other values see https://github.com/mozilla-mobile/android-components/blob/f43545fb41669904d26e1ea3546c388e2d895aed/components/feature/sitepermissions/src/test/java/mozilla/components/feature/sitepermissions/db/SitePermissionEntityTest.kt#L43

Reply:
Also we need to update this [test](https://github.com/mozilla-mobile/android-components/blob/f43545fb41669904d26e1ea3546c388e2d895aed/components/feature/sitepermissions/src/test/java/mozilla/components/feature/sitepermissions/db/SitePermissionEntityTest.kt#L48) checks when we are converting  domain to db (`convert from domain class to db entity`)